### ISSUE TITLE
parsing exception if team strength is decimal

### ISSUE THREAD

**Issue by StegSchreck** [WiSchLabs/ofm_helper#93]
models.py:403  MatchTeamStatistics
strength = models.IntegerField(default=0)

has to be changed to DecimalField

blocked by #49

**Comment by Sh4kE** [comment]
And why is it blocked by #49? :P

**Comment by StegSchreck** [comment]
same reason you fixed the country bug in that branch. because of the squashed migrations.

**Comment by Sh4kE** [comment]
It's already merged into the master ;P

**Comment by StegSchreck** [comment]
no review? ok...
